packages/measure/src/measure-renders.tsx

```diff
@@ -35,17 +35,28 @@ export async function measureRender(ui: React.ReactElement, options?: MeasureOpt
   const warmupRuns = options?.warmupRuns ?? config.warmupRuns;
 
   const { render, cleanup } = resolveTestingLibrary();
+  const testingLibrary = getTestingLibrary();
 
   showFlagsOutputIfNeeded();
 
   const runResults: RunResult[] = [];
   let hasTooLateRender = false;
+  let hasUnnecessaryRender = false;
+  let components: RenderMeasureResult = [];
   for (let i = 0; i < runs + warmupRuns; i += 1) {
     let duration = 0;
     let count = 0;
     let isFinished = false;
 
```

Comment:
detect initial rendundant rerenders:

```
// Count renders happening before `render` returns. More than 1 render means there are redundant renders for things like `useEffect()`.
if (screen == null) {
  intialRendersCount += 1;
}
```

Reply:
Actually I believe we do not need to hold `intialRendersCount ` because in the end I calculate the initial redundant renders by counting undefined values on the json array